### PR TITLE
feat: add gt, lt, gte, lte filters

### PR DESCRIPTION
**Pull request by Goodmain**
Refs: #76

### CODE REVIEW

File: src/Traits/SearchTrait.php [hunk 1]
```diff
@@ -294,6 +320,18 @@ public function filterTo(string $field, bool $strict = true, ?string $filterName
         return $this;
     }
 
+    public function filterLess(string $field, bool $strict = true, ?string $filterName = null): self
+    {
+        $filterName = empty($filterName) ? 'to' : $filterName;
+        $sign = $strict ? '<' : '<=';
```
Comment (DenTray) [inline src/Traits/SearchTrait.php:326]:
```suggestion
        $sign = ($strict) ? '<' : '<=';
```

File: src/Traits/SearchTrait.php [hunk 2]
```diff
@@ -294,6 +320,18 @@ public function filterTo(string $field, bool $strict = true, ?string $filterName
         return $this;
```
Comment (DenTray) [inline src/Traits/SearchTrait.php:320]:
```suggestion
        return $this->filterLess($field, $strict, $filterName);
```

File: src/Traits/SearchTrait.php [hunk 3]
```diff
@@ -294,6 +320,18 @@ public function filterTo(string $field, bool $strict = true, ?string $filterName
...
     }
 
+    public function filterLess(string $field, bool $strict = true, ?string $filterName = null): self
```
Comment (DenTray) [inline src/Traits/SearchTrait.php:323]:
```suggestion
    public function filterLess(string $field, bool $isStrict = true, ?string $filterName = null): self
```

File: src/Traits/SearchTrait.php [hunk 4]
```diff
@@ -282,6 +295,19 @@ public function filterFrom(string $field, bool $strict = true, ?string $filterNa
         return $this;
```
Comment (DenTray) [inline src/Traits/SearchTrait.php:295]:
```suggestion
        return $this->filterGreater($field, $strict, $filterName);
```

File: src/Traits/SearchTrait.php [hunk 5]
```diff
@@ -282,6 +295,19 @@ public function filterFrom(string $field, bool $strict = true, ?string $filterNa
...
     }
 
+    public function filterGreater(string $field, bool $strict = true, ?string $filterName = null): self
```
Comment (DenTray) [inline src/Traits/SearchTrait.php:298]:
```suggestion
    public function filterGreater(string $field, bool $isStrict = true, ?string $filterName = null): self
```

**Review by DenTray — CHANGES_REQUESTED** [review]

File: src/Traits/SearchTrait.php (hunk 1 above)
Comment (Goodmain) [reply src/Traits/SearchTrait.php]: Done

File: src/Traits/SearchTrait.php (hunk 2 above)
Comment (Goodmain) [reply src/Traits/SearchTrait.php]: Done

File: src/Traits/SearchTrait.php (hunk 3 above)
Comment (Goodmain) [reply src/Traits/SearchTrait.php]: Done

File: src/Traits/SearchTrait.php (hunk 4 above)
Comment (Goodmain) [reply src/Traits/SearchTrait.php]: Done

**Review by DenTray — APPROVED** [review]

**Pull request merged**